Very cool thank you!
Do you mind fixing the linter issues?
```
lint-check: commands[0]> flake8 pycaw/ tests/ examples/ setup.py
examples/audio_endpoint_volume_example.py:5:1: F401 'comtypes.CLSCTX_ALL' imported but unused
examples/audio_endpoint_volume_example.py:7:1: F401 'pycaw.pycaw.IAudioEndpointVolume' imported but unused
examples/list_and_switch_devices_example.py:14:[8](https://github.com/AndreMiras/pycaw/actions/runs/15078563502/job/42402445668?pr=103#step:5:9)9: E501 line too long (118 > 88 characters)
examples/volume_callback_example.py:6:1: F401 'comtypes.CLSCTX_ALL' imported but unused
examples/volume_callback_example.py:8:1: F401 'pycaw.pycaw.IAudioEndpointVolume' imported but unused
pycaw/utils.py:261:8[9](https://github.com/AndreMiras/pycaw/actions/runs/15078563502/job/42402445668?pr=103#step:5:10): E501 line too long (96 > 88 characters)
pycaw/utils.py:319:89: E501 line too long (96 > 88 characters)
lint-check: exit 1 (0.70 seconds) D:\a\pycaw\pycaw> flake8 pycaw/ tests/ examples/ setup.py pid=6148
  lint-check: FAIL code 1 (12.55=setup[[11](https://github.com/AndreMiras/pycaw/actions/runs/15078563502/job/42402445668?pr=103#step:5:12).84]+cmd[0.70] seconds)
  evaluation failed :( (12.66 seconds)
```

--- a/examples/audio_endpoint_volume_example.py
+++ b/examples/audio_endpoint_volume_example.py
@@ -2,9 +2,7 @@
 Get and set access to master volume example.
 """
 
-from comtypes import CLSCTX_ALL
-
-from pycaw.pycaw import AudioUtilities, IAudioEndpointVolume
+from pycaw.pycaw import AudioUtilities
 
 
 def main():

--- a/examples/list_and_switch_devices_example.py
+++ b/examples/list_and_switch_devices_example.py
@@ -8,7 +8,7 @@ from pycaw.utils import AudioDevice
 import warnings
 
 
-def list_active_output_devices():
+def get_active_output_devices():
     with warnings.catch_warnings():  # suppress COMError warnings
         warnings.simplefilter("ignore", UserWarning)
         return AudioUtilities.GetAllDevices(data_flow=EDataFlow.eRender.value,
@@ -26,7 +26,7 @@ def set_default_device(device: AudioDevice):
 if __name__ == "__main__":
     # List devices
     print("List of available output devices (* = default): ")
-    active_output_devices = list_active_output_devices()
+    active_output_devices = get_active_output_devices()
     default_device = get_default_device()
     other_device = None
     for device in active_output_devices:

--- a/examples/list_and_switch_devices_example.py
+++ b/examples/list_and_switch_devices_example.py
@@ -11,7 +11,8 @@ import warnings
 def list_active_output_devices():
     with warnings.catch_warnings():  # suppress COMError warnings
         warnings.simplefilter("ignore", UserWarning)
-        return AudioUtilities.GetAllDevices(data_flow=EDataFlow.eRender.value, device_state=DEVICE_STATE.ACTIVE.value)
+        return AudioUtilities.GetAllDevices(data_flow=EDataFlow.eRender.value,
+                                            device_state=DEVICE_STATE.ACTIVE.value)
 
 
 def get_default_device():

--- a/examples/list_and_switch_devices_example.py
+++ b/examples/list_and_switch_devices_example.py
@@ -43,7 +43,7 @@ if __name__ == "__main__":
 
         # List devices again
         print("Updated list of available output devices (* = default): ")
-        active_output_devices = list_active_output_devices()
+        active_output_devices = get_active_output_devices()
         default_device = get_default_device()
         for device in active_output_devices:
             if device.id == default_device.id:

--- a/examples/volume_callback_example.py
+++ b/examples/volume_callback_example.py
@@ -3,13 +3,8 @@ IAudioEndpointVolumeCallback.OnNotify() example.
 The OnNotify() callback method gets called on volume change.
 """
 
-from comtypes import CLSCTX_ALL, COMObject
-
-from pycaw.pycaw import (
-    AudioUtilities,
-    IAudioEndpointVolume,
-    IAudioEndpointVolumeCallback,
-)
+from comtypes import COMObject
+from pycaw.pycaw import AudioUtilities, IAudioEndpointVolumeCallback
 
 
 class AudioEndpointVolumeCallback(COMObject):

--- a/pycaw/utils.py
+++ b/pycaw/utils.py
@@ -258,7 +258,8 @@ class AudioUtilities:
         return AudioDevice(id, audioState, properties, dev)
 
     @staticmethod
-    def GetAllDevices(data_flow=EDataFlow.eAll.value, device_state=DEVICE_STATE.MASK_ALL.value):
+    def GetAllDevices(data_flow=EDataFlow.eAll.value,
+                      device_state=DEVICE_STATE.MASK_ALL.value):
         devices = []
         deviceEnumerator = comtypes.CoCreateInstance(
             CLSID_MMDeviceEnumerator, IMMDeviceEnumerator, comtypes.CLSCTX_INPROC_SERVER
@@ -316,4 +317,5 @@ class AudioUtilities:
         for role in roles:
             hr = policy_config.SetDefaultEndpoint(devId, role.value)
             if hr != 0:
-                raise OSError(f"SetDefaultEndpoint failed for role {role} with HRESULT {hr:#x}")
+                raise OSError(f"SetDefaultEndpoint failed for role {role} "
+                              f"with HRESULT {hr:#x}")

